table/queries/merged_reduced_scans.sql

```diff
@@ -131,6 +136,14 @@ CREATE TEMP FUNCTION ExtractServerHeader(received_headers ARRAY<STRING>) AS (
    WHERE STARTS_WITH(header, "Server: ") LIMIT 1)
 );
 
+# Error string with either a single error or multiple split by ;
+# Returns the first error
+CREATE TEMP FUNCTION GetFirstError(errors STRING) AS (
+  IF(REGEXP_CONTAINS(errors, ';'),
+     REGEXP_EXTRACT(errors, '(.*?);'),
+     errors)
+);
```

Comment:
No need to use regex, which are more expensive.

Something like this should work:
```suggestion
  IF(STRPOS(errors, ';') > 0, SUBSTR(errors, 1, STRPOS(errors, ';') - 1), errors)
```

Reference: https://cloud.google.com/bigquery/docs/reference/standard-sql/string_functions

Reply:
changed